src/Nri/Ui/SpriteSheet.elm

```diff
@@ -0,0 +1,91 @@
+module Nri.Ui.SpriteSheet exposing (checkmark, exclamationMark, bulb)
```

Comment:
❔ Do we want to expose this module so we can use these icons in monolith and monorepo directly?

Reply:
Maybe? Maybe we could wait until we need them though -- easier to expose new stuff than remove exposed stuff

Reply:
Good point!